Apps/frontend/src/views/Oscilloscope.svelte

```diff
@@ -71,6 +74,7 @@
     var startPackage = window.performance.now();
 
     let samples = new Float64Array(messageEvent.data);
+    if (isGND) {samples.fill(0.0);}
```

Comment:
Maybe use a different array. Not sure about the performance when filling the huge array with zeros.
```javascript
if (isGND) {
    var gndSample = new Array(10).fill(0.0)
    waveElement.updateBuffer(gndSample, 0, 10)
    return
}
```

This way don't waste compute time with filling the array.